ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -0,0 +1,1170 @@
+
+  // Add up the total cost for repeating_<section>:<name>TP
+  // and put it in <section>_total_TP
+  const updateTotalTP = function(section, name) {
+    if (name === undefined) { name = section }
+    getSectionIDs(section, function(ids) {
+      var TP_fields = ids.map(id => `repeating_${section}_${id}_${name}TP`);
+      getAttrs(TP_fields, function(tps) {
+        var update = {}
+        update[section + "_total_TP"] = sumValues(tps)
+        setAttrs(update);
+      });
+    });
+  }
+
+  // We record the number of CP costs of 1. This adds them all up,
+  // and subtracts as many of them from the total as breadth allows.
+  const updateDistributedCP = function() {
+    const feat = "feat_total_single_CP";
+    const skill = "skill_total_single_CP";
+    const arcane = "arcane_total_single_CP";
+    const innate = "innate_total_single_CP";
+    const tp = "TP";
+    getAttrs([feat, skill, arcane, innate, tp], 
+    function(singles) {
+      const total = (Number(singles[feat])
+              + Number(singles[skill])
+              + Number(singles[arcane])
+              + Number(singles[innate]));
+      // log("inputs: " + "'" + singles[advantage] 
+      //                + "' '" + singles[feat]
+      //                + "' '" + singles[skill]
+      //                + "' '" + singles[arcane]
+      //                + "' '" + singles[innate] + "'");
+      // log("Distributed: " + Math.min(total, Math.min(singles[tp], 6)
+      //     + "  " + total);
+      setAttrs({"distributed_CP": Math.min(total, Math.min(singles[tp], 6))});
+    });
+  }
+
+  on("change:tp" +
+      " change:feat_total_single_CP" +
+      " change:skill_total_single_CP" +
+      " change:arcane_total_single_CP" +
+      " change:innate_total_single_CP",
+      updateDistributedCP);  
+
+  // Add up the total cost for repeating_<section>:<name>CP
+  // and put it in <section>_total_CP
+  const updateTotalCP = function(section, name) {
+    if (name === undefined) { name = section }
+    getSectionIDs(section, function(ids) {
+      var CP_fields = ids.map(id => `repeating_${section}_${id}_${name}CP`);
+      getAttrs(CP_fields, function(cps) {
+        var update = {}
+        update[section + "_total_CP"] = sumValues(cps)
+        update[section + "_total_single_CP"] = sumSingleValues(cps)
+        setAttrs(update);
+      });
+    });
+  }
+
+
+  var getSectionIDsOrdered = function (sectionName, callback) {
+    const order_name = `_reporder_repeating_${sectionName}`
+    getAttrs([order_name], function (v) {
+      getSectionIDs(sectionName, function (idArray) {
+        let reporderArray = (v[order_name] ?
+                            v[order_name].toLowerCase().split(',') :
+                            []),
+            still_present = reporderArray.filter(x => idArray.includes(x))
+        callback([...new Set(still_present.concat(idArray))]);
+      });
+    });
+  };
+
+  // ----- Top action and roll -----
+
+  var toggleSavePending = function() {
```

Comment:
For example, the linter catches these `var`'s:
![image](https://user-images.githubusercontent.com/5629800/208280608-1701ef30-c8c3-41b2-9cb8-56f40a47acf5.png)